dev/ci/nix/default.nix

```diff
@@ -20,8 +20,16 @@ let mathcomp = coqPackages.mathcomp.overrideAttrs (o: {
 let ssreflect = coqPackages.ssreflect.overrideAttrs (o: {
   inherit (mathcomp) src;
   }); in
-let coq-ext-lib = coqPackages.coq-ext-lib; in
-let simple-io = coqPackages.simple-io; in
+
+let coq-ext-lib = coqPackages.coq-ext-lib.overrideAttrs (o: {
+    src = fetchTarball "https://github.com/coq-ext-lib/coq-ext-lib/tarball/master";
+  }); in
+
+let simple-io = coqPackages.simple-io.overrideAttrs (o: {
+    src = fetchTarball "https://github.com/Lysxia/coq-simple-io/tarball/master";
+    buildInputs = o.buildInputs ++ [ coq-ext-lib ];
```

Comment:
Why do you need to change this. Is it because the last packaged version did not depend on coq-ext-lib? Can we update the version in nixpkgs to avoid this change?

Reply:
Yes we can. See: https://github.com/NixOS/nixpkgs/pull/57708

Reply:
Cool!

Reply:
Fixed.